lms/djangoapps/commerce/tests.py

```diff
@@ -245,3 +251,14 @@ def test_no_settings(self):
         # Ensure that the user is not enrolled and that no calls were made to the E-Commerce API
         self.assertTrue(CourseEnrollment.is_enrolled(self.user, self.course.id))
         self.assertIsInstance(httpretty.last_request(), HTTPrettyRequestEmpty)
+
+    @httpretty.activate
+    def test_empty_sku(self):
+        """ If the CourseMode has an empty string for a SKU, the API should not be used. """
+
```

Comment:
Nitpick: could you remove this newline, to be more consistent with the other test cases?


What? I had to find something to comment on...